docker/object_detector/Dockerfile

```diff
@@ -0,0 +1,71 @@
+##############################################################################
+# Base system packages and depedencies
+#
+FROM nvidia/cuda:9.2-cudnn7-runtime-ubuntu18.04 AS ptg-gpu
+CMD nvidia-smi
```

Comment:
This intermediate build doesn't seem to be used or referenced elsewhere? Am I missing it?